cns/restserver/util.go

```diff
@@ -616,6 +616,37 @@ func (service *HTTPRestService) getNetworkContainerDetails(networkContainerID st
 	return containerDetails, containerExists
 }
 
+func (service *HTTPRestService) getAllNetworkContainersDetails(networkContainersID []string) (map[string]cns.ContainerStatus, bool) {
+	service.RLock()
+	defer service.RUnlock()
+
+	allNcExist := true
+	containerDetails := make(map[string]cns.ContainerStatus)
+
+	for _, ncID := range networkContainersID {
+		ncDetails, ncExists := service.state.ContainerStatus[ncID]
+		if !ncExists {
+			allNcExist = false
+			break
+		}
+
```

Comment:
nit: tighten up, remove extra vertical whitespace. It is okay to separate logical blocks, but use empty lines very very sparingly.

Reply:
Fixed.